lib/dogapi/version.rb

```diff
@@ -1,3 +1,3 @@
 module Dogapi
-  VERSION = '1.37.1'
+  VERSION = '1.37.2'
```

Comment:
No need to change this while creating features, it'll get bumped at release time 😄